frontend/public/components/project-overview.jsx

```diff
@@ -10,42 +10,45 @@ import { ResourceLink, resourcePath } from './utils';
 
 export const ComponentLabel = ({text}) => <div className="co-component-label">{text}</div>;
 
-const ProjectOverviewListItem = ({obj, onClick, selectedItem}) => {
-  const {currentController, kind, metadata} = obj;
-  const {namespace, name, uid} = metadata;
-  const isSelected = obj.metadata.uid === _.get(selectedItem, 'metadata.uid', '');
+const ProjectOverviewListItem = ({item, onClick, selectedItem}) => {
+  const {controller, readiness, resource} = item;
+  const {namespace, name, uid} = resource.metadata;
+  const isSelected = resource.metadata.uid === _.get(selectedItem, 'resource.metadata.uid', '');
   const className = classnames('project-overview__item', {'project-overview__item--selected': isSelected});
   const heading = <h3 className="project-overview__item-heading">
     <ResourceLink
       className="co-resource-link-truncate"
-      kind={kind}
+      kind={resource.kind}
       name={name}
       namespace={namespace}
     />
   </h3>;
 
   const additionalInfo = <div key={uid} className="project-overview__additional-info">
-    { currentController &&
+    { controller &&
       <div className="project-overview__detail project-overview__detail--controller">
-        <ComponentLabel text={_.startCase(currentController.kind)} />
+        <ComponentLabel text={_.startCase(controller.resource.kind)} />
         <ResourceLink
           className="co-resource-link-truncate"
-          kind={currentController.kind}
-          name={currentController.metadata.name}
+          kind={controller.resource.kind}
+          name={_.get(controller, 'resource.metadata.name')}
           namespace={namespace}
         />
       </div>
     }
-    <div className="project-overview__detail project-overview__detail--status">
-      <ComponentLabel text="Status" />
-      <Link to={`${resourcePath(kind, name, namespace)}/pods`}>
-        {obj.status.replicas || 0} of {obj.spec.replicas} pods
-      </Link>
-    </div>
+    {
+      readiness &&
```

Comment:
We should think about how we can expand these rows for other types that don't have pods (like service instances) later.

Nothing needed for this PR, just an observation.